app/views/code_reviews/_copy_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for code_review, url: code_review_copy_index_path, html: { method: :post } do |f| %>
```

Comment:
Updated. It was a result of my incorrect use of the singular for the controller name, as specified below.